var/spack/repos/builtin/packages/rocm-openmp-extras/package.py

```diff
@@ -41,7 +42,8 @@
     "50e9e87ecd6b561cad0d471295d29f7220e195528e567fcabe2ec73838979f61",
     "83ed7aa1c9322b4fc1f57c48a63fc7718eb4195ee6fde433009b4bc78cb363f0",
     "49cfa8f8fc276ba27feef40546788a2aabe259a924a97af8bef24e295d19aa5e",
-    "47dbcb41fb4739219cadc9f2b5f21358ed2f9895ce786d2f7a1b2c4fd044d30f"
+    "47dbcb41fb4739219cadc9f2b5f21358ed2f9895ce786d2f7a1b2c4fd044d30f",
+    "c41958560ec29c8bf91332b9f668793463904a2081c330c0d828bf2f91d4f04e",
```

Comment:
Not a deal breaker here, but there is an extra comma on the last hash.

Reply:
thx for noticing it ..correcting it